package.json

```
@@ -37,6 +37,7 @@
     "aws-xray-sdk-core": "3.4.1",
     "aws-xray-sdk-express": "3.4.1",
     "cors": "2.8.5",
+    "entities": "^4.5.0",
```

```suggestion
    "entities": "4.5.0",
```
so renovate can manage

---

I think Renovate bot will pin a new package automatically to a patch version when there's an update, so this could be left as-is